app/views/form/_checkbox_question.html.erb

```diff
@@ -4,14 +4,17 @@
     caption: caption(caption_text, page_header, conditional),
     legend: legend(question, page_header, conditional),
     hint: { text: question.hint_text&.html_safe } do %>
+    <% after_divider = false %>
 
     <% question.answer_options.map do |key, val| %>
       <% if key.starts_with?("divider") %>
-        <%= f.govuk_check_box_divider %>
+      <% after_divider = true %>
```

Comment:
Interesting approach! Is it definitely always the case that anything separated by a divider is mutually exclusive? @paulrobertlloyd? If so, then :very_nice:

Reply:
Hmm, well our dividers put "or" between the options, so I'd assume so 👀

Reply:
Yup, we’d want to separate out those options that are mutually exclusive after a divider. See https://design-system.service.gov.uk/components/checkboxes/#add-an-option-for-none-

Reply:
Whether we’d use a divider for other use cases with checkboxes I don’t know, but I suspect not. This is a good assumption to go forward with, for now.